public/Students/Leo Jayachandran/DynamicIndex/dynamicIndexMaker.js

```diff
@@ -253,7 +278,13 @@
 				let $termText = $(`<p>${alphabetisedIndex[i].terms[u].name}</p>`);
 				let $pagesText = $(`<div class = "pagesTextDiv"></div>`);
 				for (let j = 0; j < alphabetisedIndex[i].terms[u].pages.length; j++) {
-					$pagesText.append(`<a class = 'indexPages' title = "${alphabetisedIndex[i].terms[u].pages[j].pageName}" href = "${alphabetisedIndex[i].terms[u].pages[j].pageLink}">${alphabetisedIndex[i].terms[u].pages[j].pageName}</a> &#10; <br/>`);
+                    var pageToAdd = $(`<a></a>`).html(alphabetisedIndex[i].terms[u].pages[j].pageName).attr({
+                        "title": alphabetisedIndex[i].terms[u].pages[j].pageName,
+                        "href": alphabetisedIndex[i].terms[u].pages[j].pageLink,
+                        "class" : "indexPages"
+                    });
+
+					$pagesText.append(pageToAdd, `&#10; <br/>`);
```

Comment:
I like your appropriate usage of $JQuery variables. It makes it very clear when you are using JQuery objects.